ui/app/AppLayouts/Wallet/popups/AddAccountWithPrivateKeyModal.qml

```diff
@@ -81,17 +81,28 @@ ModalPopup {
         validationError: popup.passwordValidationError
     }
 
-
     StyledTextArea {
         id: accountPKeyInput
+        customHeight: 88
         anchors.top: passwordInput.bottom
         anchors.topMargin: marginBetweenInputs
+        anchors.left: parent.left
+        anchors.right: parent.right
+        validationError: popup.privateKeyValidationError
+        //% "Private key"
```

Comment:
Can remove this, or maybe the label needs to be added back?

Reply:
Third option, push :facepalm: label added back. thanks